ml/Config.cc

```diff
@@ -80,6 +111,10 @@ void Config::readMLConfig(void) {
     ADWindowRateThreshold = clamp(ADWindowRateThreshold, 0.01, 0.99);
     ADDimensionRateThreshold = clamp(ADDimensionRateThreshold, 0.01, 0.99);
 
+    SaveAnomalyPercentageEvery = clamp(SaveAnomalyPercentageEvery, 60.0, 240.0);
```

Comment:
When picking up the value of `SaveAnomalyPercentageEvery` from the config (line 50 above) the default value is indicated to be `15 * 60`, but this, does it set it to a maximum value of 240 seconds, vs 915?

Also, should this value be configurable at all ? I mean does the user have any reason to put something else there?

Reply:
Thanks, will change the max value, we decided to leave it configurable for POC the least, but your point is valid for MVP. This is a question of resolution, rather similar to few configurable training algorithm parameters.